model/minsky.cc

```diff
@@ -352,8 +354,8 @@ namespace minsky
        // ensure that initial values of pasted parameters are correct. for ticket 1258
        if (auto v=i->variableCast())
 		 if (v->type()==VariableType::parameter && !existingParms.empty()) 
-		   for (vector<pair<string,string>>::iterator it = existingParms.begin() ; it != existingParms.end(); ++it)
-		      if (v->valueId()==(*it).first) v->init((*it).second);
+		   for (auto& it: existingParms)
```

Comment:
Please use "auto it=existingParms.find(v->valueId(); if (it!=existingParms.end()) ..."

Reply:
Done :)